framework/src/problems/FEProblemBase.C

```diff
@@ -5872,7 +5891,12 @@ FEProblemBase::checkExceptionAndStopSolve(bool print_message)
 
     // Print the message
     if (_communicator.rank() == 0 && print_message)
-      Moose::err << _exception_message << std::endl;
+    {
+      _console << "\nThe following exception was caught in FEProblemBase:\n";
```

Comment:
Note that this was going to `Moose::err` before, but that made the output weird. I think the standard console output is where we want this.